dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -37,20 +37,25 @@ public class StepwisePlanner
     public StepwisePlanner(
         IKernel kernel,
         StepwisePlannerConfig? config = null,
-        string? prompt = null)
+        string? prompt = null,
+        PromptTemplateConfig? promptUserConfig = null)
```

Comment:
`promptUserConfig` requires a matching XML tag in the method comments above. (PR checks are blocking on this)

Reply:
Thanks. Added